common/djangoapps/enrollment/views.py

```diff
@@ -455,6 +456,18 @@ def post(self, request):
                 # Will reactivate inactive enrollments.
                 response = api.add_enrollment(username, unicode(course_id), mode=mode, is_active=is_active)
 
+            if has_api_key_permissions:
```

Comment:
nit: could you add an inline comment to make explicit the assumption we're making about this permission?